### ISSUE TITLE
[Bug]: [bulkload] error msg needs improvement if import empty json file 

### ISSUE THREAD

**Issue by yanliang567** [milvus-io/milvus#16899]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:master-20220510-2d0f908d
- Deployment mode(standalone or cluster): standalone
- SDK version(e.g. pymilvus v2.0.0rc2): pymilvus-2.1.0.dev54
```

### Current Behavior

bulk load data with column based, and collection schema auto_id=False:
```
[2022-05-11 10:08:43,852 - DEBUG - ci_test]: (api_response) : <Bulk load state:
    - taskID    : 433123471866462209,
    - state     : BulkLoadFailed,
    - row_count : 0,
    - infos     : {'files': 'col_cust_float_vectors_only_128d_0_0.json', 'failed_reason': 'JSON parse: EOF'}

```

### Expected Behavior

if collection schema auto_id=True, import the same empty json file, the error msg makes more sense:
```
[2022-05-11 10:08:42,588 - DEBUG - ci_test]: (api_response) : <Bulk load state:
    - taskID    : 433123471525675009,
    - state     : BulkLoadFailed,
    - row_count : 0,
    - infos     : {'files': 'col_auto_float_vectors_only_128d_0_0.json', 'failed_reason': 'JSON column consumer: row count is 0'}
```

### Steps To Reproduce

```markdown
1. prepare an empty json file with only below content:
   {}
2. create collection with auto_id=False
3. import the empty json
```

**Comment by yhmo** [comment]
Fixed by #16958

**Comment by yanliang567** [comment]
verified error msg: "JSON parse: row count is 0" on master-20220513-52fc9a55
